frontend/src/components/annotation/audio_annotation/render_elements/waveform_selector.vue

```diff
@@ -53,12 +53,17 @@ export default Vue.extend({
       zoom: 0,
       regions: null,
       disableDragSelection: null,
+      // NOTE: regionIdToIsCreatedMap is used to keep track of rendered regions
+      regionIdToIsCreatedMap: {},
```

Comment:
Added `regionIdToIsCreatedMap` to detect already rendered regions, so multiple rendering of the same annotations can be avoided